src/rest-server/src/models/user.js

```diff
@@ -259,7 +259,7 @@ if (config.env !== 'test') {
         if (errMsg) {
           logger.warn('get user list failed', errMsg);
         } else {
-          logger.warn('users:', userInfoList);
+          // logger.warn('users:', userInfoList);
```

Comment:
why annotate this?

Reply:
do not need to list all users in log, seems it was a debug log